pkg/prometheus/promcfg.go

```diff
@@ -16,6 +16,7 @@ package prometheus
 
 import (
 	"fmt"
+	"github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1alpha1"
```

Comment:
(nit) this would need to go with the other prometheus-operator/prometheus-operator imports.

Reply:
Moved it in 3f4fc7a but I see we have the following there:
```
	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
	v1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
	"github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1alpha1"
```

What should the correct aliases be?

Reply:
no strong opinion, `monitoringv...` is probably clearer though more verbose.